week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,8 +7,14 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
+function convertToUSD(price) {
+  var result = price * 1.4 + 0.01 + " $ ";
```

Comment:
Close! Instead of adding `+ 0.01` you need to multiple the price by it: `* 0.01`. That would look like:

```
var result = (price * 0.01) * 1.4 + " $";
```

The same is true for the function below.